MapboxNavigationTests/CarPlayManagerTests.swift

```diff
@@ -431,6 +431,10 @@ class CarPlayManagerSpec: QuickSpec {
         func carPlayManagerDidEndNavigation(_ carPlayManager: CarPlayManager) {
             //no-op
         }
+        
+        func carPlayManager(_ carPlayManager: CarPlayManager, navigationServiceAlong route: Route) -> NavigationService {
+            return MapboxNavigationService(route: route)
```

Comment:
This might be fine but I wonder if we should be using a dummy instead? Init'ing this class does have some side effects.